packages/expo-cli/src/commands/client/index.js

```diff
@@ -0,0 +1,101 @@
+import chalk from 'chalk';
+import CliTable from 'cli-table';
+import { Credentials, User, ApiV2 } from 'xdl';
+
+import urlOpts from '../../urlOpts';
+import * as appleApi from '../build/ios/appleApi';
+import { runAction, travelingFastlane } from '../build/ios/appleApi/fastlane';
+import * as credentials from '../build/ios/credentials';
+import promptForCredentials from '../build/ios/credentials/prompt/promptForCredentials';
+import selectDistributionCert from './selectDistributionCert';
+import selectPushKey from './selectPushKey';
+import generateBundleIdentifier from './generateBundleIdentifier';
+import log from '../../log';
+import prompt from '../../prompt';
+
+export default program => {
+  program
+    .command('client:ios')
+    .option(
+      '--apple-id <login>',
+      'Apple ID username (please also set the Apple ID password as EXPO_APPLE_PASSWORD environment variable).'
+    )
+    .asyncAction(async options => {
```

Comment:
I prefer to keep command files clean - so that they only contain the definition of a command and the "business logic" is kept in a separate file. Consider this as a suggestion, I don't have a strong opinion on this.